Weather/Views/TodayWeatherView.swift

```diff
@@ -12,7 +12,7 @@ struct TodayWeatherView: View {
     
     var body: some View {
         VStack{
-            Text("Today")
+            Text(NSLocalizedString("Today", comment: ""))
```

Comment:
There's no need to use `NSLocalizedString` in SwiftUI, you can just write `Text("Today")` or you should use `LocalizedStringKey`